source/_integrations/zha.markdown

```diff
@@ -316,6 +316,20 @@ To add a new device:
 1. Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g., turn on/off lights up to 10 times, switches usually have a reset button/pin). It might take a few seconds for the devices to appear. You can click on **Show logs** for more verbose output.
 1. Once the device is found, it will appear on that page and will be automatically added to your devices. You can optionally change its name and add it to an area (you can change this later). You can search again to add another device, or you can go back to the list of added devices.
 
+#### Best practices for avoiding pairing difficulties
+
+- If possible try to pair your Zigbee devices in their intended final location, (and not pair it next to the Zigbee coordinator and then need to move it after). 
+  - Pairing a Zigbee device next to the Zigbee coordinator and then moving it later can result in dropped/lost connections or other issues. 
+- Zigbee devices that in the past have been connected to another Zigbee coordinator (hub/bridge/gateway/controller) can usually only be added/paired in ZHA after they have been restored to their factory reset settings. 
```

Comment:
```suggestion
```

https://deploy-preview-17466--home-assistant-docs.netlify.app/integrations/zha/#add-philips-hue-bulbs-that-have-previously-been-added-to-another-bridge

This is already documented on the same page.